src/core_init_atmosphere/mpas_init_atm_static.F

```diff
@@ -282,13 +283,17 @@ subroutine init_atm_static(mesh, dims, configs)
 !
 ! Set land use and soil category parameters for water and ice
 !
+! TODO: Should this be moved to be above the interpolation of land use and soil at? Or
+! is it fine here?
```

Comment:
This should be left here (so we can throw an error as soon as possible if the user specified a bad value for `config_landuse_data`), but we should move setting `isice_lu`, `iswater_lu` and `ismax_lu` into the `geotile_manager`.